tests/conftest.py

```diff
@@ -29,7 +29,8 @@
 from .mock_clients.mock_s3 import mocked_session_class_factory
 
 
-load_dotenv(find_dotenv())
+if os.getenv("USE_LIVE_CLOUD") == "1":
+    load_dotenv(find_dotenv())
```

Comment:
This had fooled my local testing by loading environment variables when I wasn't expecting them for the mocked tests.